client/src/components/layouts/DashboardSidebarData.js

```diff
@@ -8,8 +8,8 @@ import {
   BsBookHalf,
   BsPerson,
   BsPeople,
-  BsGearFill,
-  BsFillQuestionCircleFill,
+  // BsGearFill,
+  // BsFillQuestionCircleFill,
```

Comment:
remove if not needed

Reply:
This part will have to confirm again with others. Not sure if we still need `setting` and `help` pages. Will remove if we don't need them anymore.

Reply:
SGTM! @pmyip3 could you give some insight into this?

Reply:
@mable-yip

Reply:
Just confirmed with Mable, we're keeping the `help button` as commented for now, but removing the `setting button`, the code is updated.